client/src/components/Login.js

```diff
@@ -1,13 +1,28 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { Button, Form } from "semantic-ui-react";
 import { Redirect } from "react-router-dom";
+import { getProfileByApplicantId } from "../api/profiles";
 import { signApi } from "../api/auth";
 
 const Login = () => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const [error, setError] = useState(null);
   const [redirectRoute, setRedirectRoute] = useState("");
+  const [applicantId, SetApplicantId] = useState(null);
+
+  useEffect(() => {
+    const fetchProfile = async applicantId => {
+      const response = await getProfileByApplicantId(applicantId);
+      if (Object.keys(response).length !== 0) {
```

Comment:
After changing the API endpoint you can update this to `response.length` without using Object.keys

Reply:
Changed. Thanks for your assistance.